packages/@tinacms/cli/src/cmds/start-server/index.ts

```diff
@@ -42,6 +42,7 @@ interface Options {
   command?: string
   watchFolders?: string[]
   noWatch?: boolean
+  noServer?: boolean
```

Comment:
we should confirm with @jeffsee55, but I believe the noServer option was related to the file locking issues when using ClassicLevel, which should no longer be an issue with many-level, so we may want to drop this change.

Reply:
Remvoed